requirements.txt

```diff
@@ -0,0 +1,12 @@
+astroid==2.4.2
+autopep8==1.5.3
+colorama==0.4.3
+isort==4.3.21
+lazy-object-proxy==1.4.3
+mccabe==0.6.1
+pycodestyle==2.6.0
+pylint==2.5.3
```

Comment:
I added the `pylint` linter which is a dependency that tells you where you have syntactical or stylistic errors or warnings to help keep your code clean! It also installed some of the other dependencies listed here which are its dependencies